api/notd/manager.py

```diff
@@ -258,10 +262,31 @@ async def retreive_token_metadata(self, registryAddress: str, tokenId: str) -> R
             raise NotFoundException()
         return tokenMetadatas[0]
 
+    async def update_collection(self, address: str, tokenId: str) -> None:
+        collections = await self.retriever.list_token_metadata(
+            fieldFilters=[
+                StringFieldFilter(fieldName=TokenCollectionsTable.c.address.key, eq=address),
+            ], limit=1,
+        )
+        collection = collections[0] if len(collections) > 0 else None
+        if collection and collection.updatedDate >= date_util.datetime_from_now(days=-3):
```

Comment:
i would change this to 7 maybe - it should probably be less frequent than tokens